libs/classifiers/edgetpu/face_mask.py

```diff
@@ -50,35 +34,35 @@ def __init__(self, config):
         self.input_details = self.interpreter.get_input_details()
         self.output_details = self.interpreter.get_output_details()
 
-
-    def inference(self, resized_rgb_image):
+    def inference(self, resized_rgb_image) -> list:
```

Comment:
If this parameter is an array why don't we change the name to `resized_rgb_images` or something like that?

Reply:
It's an array of resized images. I changed its name to resized_rgb_images